doc/modules/learning_curve.rst

```diff
@@ -187,3 +187,6 @@ to :func:`learning_curve` to generate and plot the learning curve:
       X, y = shuffle(X, y, random_state=0)
       LearningCurveDisplay.from_estimator(
          SVC(kernel="linear"), X, y, train_sizes=[50, 80, 110], cv=5)
+
+For a detailed example of using learning curves, see 
```

Comment:
Could you make an "Examples" section and list the reference under that section? You can check other files in the "modules" folder for the correct implementation.

Reply:
Couldn't find the docstrings of LearningCurveDisplay class to also reference it there. Could you point me to which directory it is present in.

Reply:
Yes, it's here: https://github.com/scikit-learn/scikit-learn/blob/74a33757c/sklearn/model_selection/_plot.py#L188